style.css

```diff
@@ -75,3 +94,163 @@ header nav {
   list-style-type: none;
   margin-left: 20px;
 }
+
+.cards_container {
+  background-color: white;
+  border-radius: 20px;
+  border: 2px solid rgb(211, 211, 211);
+  padding: 30px;
+  margin: 200px 30px 30px 30px;
+}
+
+.cards_container .card_img img {
+  width: 100%;
+  height: auto;
+}
+
+.card_text {
+  font-family: 'Poppins', sans-serif;
+}
+
+.card_text .card_paragraph {
+  font-family: 'Poppins', sans-serif;
+  font-size: 25px;
+}
+
+.card_text_flex {
+  display: flex;
+  margin-left: -20px;
+}
+
+.card_lang_flex {
+  display: flex;
+  margin-left: -20px;
+}
+
+.card_button button {
+  color: #6070ff;
+  border: 1px solid #6070ff;
+  padding: 20px;
+  font-weight: bold;
+  border-radius: 10px;
+  margin: 20px 0 20px 0;
+}
+
+.about_section {
+  padding: 200px 30px 200px 30px;
+  background-color: white;
+  border-top-right-radius: 100px;
+  margin-top: 200px;
+}
```

Comment:
Please ensure that the content in this section `(about_section)` is aligned with flexbox according to the project requirement.